packages/platform-browser/src/dom/events/key_events.ts

```diff
@@ -8,57 +8,14 @@
 
 import {DOCUMENT, ɵgetDOM as getDOM} from '@angular/common';
 import {Inject, Injectable, NgZone} from '@angular/core';
+
 import {EventManagerPlugin} from './event_manager';
 
 /**
  * Defines supported modifiers for key events.
  */
 const MODIFIER_KEYS = ['alt', 'control', 'meta', 'shift'];
```

Comment:
I guess we can drop this const as well and just use `Object.keys(MODIFIER_KEY_GETTERS)` in the code? Or use it like this (so we do this once):
```suggestion
const MODIFIER_KEYS = Object.keys(MODIFIER_KEY_GETTERS);
```

Reply:
Changing this to use `Object.keys` causes lint issues of `no-toplevel-property-access`. So I've just left the array as is for now.